lib/Runtime/Types/PathTypeHandler.cpp

```diff
@@ -1881,7 +1881,7 @@ namespace Js
     }
 
     DynamicType *
-    PathTypeHandlerBase::CreateNewScopeObject(ScriptContext *scriptContext, DynamicType *type, const PropertyIdArray *propIds, PropertyAttributes extraAttributes, uint extraAttributesSlotCount)
+    PathTypeHandlerBase::CreateNewScopeObject(ScriptContext *scriptContext, DynamicType *type, const PropertyIdArray *propIds, PropertyAttributes extraAttributes, uint extraAttributesSlotCount, bool skipLetPropForArgumentsSymbol)
```

Comment:
can we have the templated arguments as bool instead of a  param. Not sure how hot this code path.